app/controllers/search_terms_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SearchTermsController < ApplicationController
+  def index
+  end
+  
+  # GET /search_terms
+  def index
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/1d081ecd36c48dd47f0123e7164aa066b7a4d685/app/controllers/search_terms_controller.rb#L6"></span>Method <code>SearchTermsController#index</code> is defined at both app/controllers/search_terms_controller.rb:2 and app/controllers/search_terms_controller.rb:6.</td>
    </tr>
  </tbody>
</table>